docs/maintain/elasticsearch.rst

```diff
@@ -97,20 +97,9 @@ Upgrade
 
 If the `ReadOnlyREST plugin <https://readonlyrest.com>`__ is used:
 
-#. Get the ReadOnlyREST plugin's ZIP file:
+#. Check the `changelog <https://github.com/beshu-tech/readonlyrest-docs/blob/master/changelog.md>`__ for a new version of ReadOnlyREST. Note which versions of Elasticsearch are supported.
 
-   #. Open the `download page <https://readonlyrest.com/download/>`__
-   #. Select "Free Elasticsearch plugin" from the *Select Product* dropdown
-   #. Select the Elasticsearch version from the *Elastic Stack Version* dropdown
-   #. Enter your email address in *Send to email*
-   #. Check *Notify me about new versions and security fixes*
-   #. Click the *Get it now* button
-
-   .. note::
-
-      A new version might not yet be available for download. You can check the `changelog <https://github.com/beshu-tech/readonlyrest-docs/blob/master/changelog.md>`__.
-
-#. Move the ZIP file to the ``salt/private/files`` directory.
+#. Update ``readonlyrest_version`` and ``elasticsearch_version`` in the ``salt/elasticsearch/plugins/readonlyrest.sls`` file
```

Comment:
When should `elasticsearch_version` be updated? The Elasticsearch version is controlled by `elasticsearch/init.sls` and whatever upgrades are then performed. Shouldn't the instruction here be to update `elasticsearch_version` to match whatever version is installed (with maybe a helpful sample command for determining the Elasticsearch version)?

Reply:
See new commit